src/index.html

```diff
@@ -4,9 +4,402 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="" class="header__logo">
+        <img src="./images/logo.png" alt="">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link">
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link">
+              СМАРТФОНЫ И ТЕЛЕФОНЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartfonu"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              ГАДЖЕТЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link">
+              ПЛАНШЕТЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link">
+              ФОТО
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link">
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="catalog__card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt=""
+            class="catalog__card_image"
+          >
+          <div class="catalog__card_name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </div>
+          <div class="catalog__card_code">
+            Код товара: 195434
+          </div>
+          <div class="catalog__card_reting"></div>
```

Comment:
```suggestion
          <div class="card__rating"></div>
```
why make this empty div at all?